src/fe-text/gui-readline.c

```diff
@@ -1174,6 +1174,7 @@ void gui_readline_init(void)
 	key_bind("key", NULL, "^H", "backspace", (SIGNAL_FUNC) key_combo);
 	key_bind("key", NULL, "^?", "backspace", (SIGNAL_FUNC) key_combo);
 	key_bind("key", NULL, "^I", "tab", (SIGNAL_FUNC) key_combo);
+	key_bind("key", NULL, "meta2-Z", "shifttab", (SIGNAL_FUNC) key_combo);
```

Comment:
rename this to `stab` to stay in line with the existing `chome` and `mup`

Reply:
ah yeah ofc, done